### PR TITLE
xteve role updated

### PR DESCRIPTION
**Pull request by satzisa**

### CODE REVIEW

File: roles/xteve/tasks/subtasks/migration.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#########################################################################
+---
+- name: Check folder exists
+  stat:
+    path: /opt/xteve
+  register: folder
+
+- name: Check subfolder exists
+  stat:
+    path: /opt/ombi/app
```
Comment (Migz93) [inline roles/xteve/tasks/subtasks/migration.yml]: Correction to reference of Ombi please.

Comment (satzisa) [reply roles/xteve/tasks/subtasks/migration.yml]: Thanks for pointing out migz. Corrected the same.

Comment (Migz93) [reply roles/xteve/tasks/subtasks/migration.yml]: Thanks for that

**Comment by Migz93** [comment]
This looks fine to me, tested it and it deploys and loads fine. Don't have any IPTV anymore to test it works fully though. Will need someone else to come along and review it too before accepting the PR.

I noticed the subdomain still doesn't take you directly into xteve though, could you do me a favour and do the following, (replace yourdomain.tld with your actual domain)
`nano /opt/nginx-proxy/vhost.d/xteve.yourdomain.tld`
And then inside this file put
```
location = / {
  return 301 /web;
}
```
And finally
`docker restart nginx-proxy`

This should redirect any xteve.yourdomain.tld requests to xteve.yourdomain.tld/web, i was able to test that fine but as I don't have IPTV i wasn't able to test that plex still streams fine.

**Comment by satzisa** [comment]
> 
> 
> This looks fine to me, tested it and it deploys and loads fine. Don't have any IPTV anymore to test it works fully though. Will need someone else to come along and review it too before accepting the PR.
> 
> I noticed the subdomain still doesn't take you directly into xteve though, could you do me a favour and do the following, (replace yourdomain.tld with your actual domain)
> `nano /opt/nginx-proxy/vhost.d/xteve.yourdomain.tld`
> And then inside this file put
> 
> ```
> location = / {
>   return 301 /web;
> }
> ```
> 
> And finally
> `docker restart nginx-proxy`
> 
> This should redirect any xteve.yourdomain.tld requests to xteve.yourdomain.tld/web, i was able to test that fine but as I don't have IPTV i wasn't able to test that plex still streams fine.

That's a good one. yea after creating the virtualhost it's redirected to xteve.domain.ltd/web. 
It works fine for me.

**Pull request merged**